### ISSUE TITLE
Fields’ isEmpty() method has been deprecated. Use isValueEmpty() instead.

### ISSUE THREAD

**Issue by Saboteur777** [craftcms/cms#2616]
### Description

Got a deprecation error in CP: 

> Fields’ isEmpty() method has been deprecated. Use isValueEmpty() instead.

origin: 
/var/www/domain.com**/vendor/craftcms/cms/src/base/Element.php:1037**

### Steps to reproduce

1. Install RC15

### Additional info

- Craft version: RC15
- PHP version: 7.2.2
- Database driver & version: MySQL 5.5.5

**Comment by rjgux** [comment]
Also seeing this, stack trace -

```

27 | Deprecation error: Fields’ isEmpty() method has been deprecated. Use isValueEmpty() instead. Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/base/Field.php:315
-- | --
craft\redactor\Field::isValueEmpty(craft\redactor\FieldData, craft\elements\MatrixBlock) Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/base/Element.php:1037
craft\elements\MatrixBlock::isFieldEmpty("text") Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/base/Element.php:776
craft\elements\MatrixBlock::__call("isFieldEmpty:text", [craft\redactor\FieldData])
call_user_func([craft\elements\MatrixBlock, "isFieldEmpty:text"], craft\redactor\FieldData) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/validators/Validator.php:444
yii\validators\RequiredValidator::isEmpty(craft\redactor\FieldData) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/validators/RequiredValidator.php:71
yii\validators\RequiredValidator::validateValue(craft\redactor\FieldData) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/validators/Validator.php:281
yii\validators\RequiredValidator::validateAttribute(craft\elements\MatrixBlock, "field:text") Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/validators/Validator.php:267
yii\validators\RequiredValidator::validateAttributes(craft\elements\MatrixBlock, ["field:text"]) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/base/Model.php:367
craft\elements\MatrixBlock::validate() Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/fields/Matrix.php:527
craft\fields\Matrix::validateBlocks(craft\elements\Entry, null) Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/base/Element.php:1019
craft\elements\Entry::validateCustomFieldAttribute("field:projectContentBlocks", [craft\fields\Matrix, "validateBlocks", null], yii\validators\InlineValidator)
call_user_func([craft\elements\Entry, "validateCustomFieldAttribute"], "field:projectContentBlocks", [craft\fields\Matrix, "validateBlocks", null], yii\validators\InlineValidator) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/validators/InlineValidator.php:72
yii\validators\InlineValidator::validateAttribute(craft\elements\Entry, "field:projectContentBlocks") Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/validators/Validator.php:267
yii\validators\InlineValidator::validateAttributes(craft\elements\Entry, ["field:projectContentBlocks"]) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/base/Model.php:367
craft\elements\Entry::validate() Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/services/Elements.php:373
craft\services\Elements::saveElement(craft\elements\Entry) Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/controllers/EntriesController.php:539
craft\controllers\EntriesController::actionSaveEntry()
call_user_func_array([craft\controllers\EntriesController, "actionSaveEntry"], []) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/base/InlineAction.php:57
yii\base\InlineAction::runWithParams(["p" => "admin/entries/projects/6-environmental-justice-foundation"]) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/base/Controller.php:157
craft\controllers\EntriesController::runAction("save-entry", ["p" => "admin/entries/projects/6-environmental-justice-foundation"]) Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/web/Controller.php:74
craft\controllers\EntriesController::runAction("save-entry", ["p" => "admin/entries/projects/6-environmental-justice-foundation"]) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/base/Module.php:528
craft\web\Application::runAction("entries/save-entry", ["p" => "admin/entries/projects/6-environmental-justice-foundation"]) Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/web/Application.php:237
craft\web\Application::runAction("entries/save-entry", ["p" => "admin/entries/projects/6-environmental-justice-foundation"]) Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/web/Application.php:445
craft\web\Application::_processActionRequest(craft\web\Request) Called from /Users/richard/repos/mud-2018/deploy/vendor/craftcms/cms/src/web/Application.php:221
craft\web\Application::handleRequest(craft\web\Request) Called from /Users/richard/repos/mud-2018/deploy/vendor/yiisoft/yii2/base/Application.php:386
craft\web\Application::run() Called from /Users/richard/repos/mud-2018/deploy/public/index.php:28


```

**Comment by brandonkelly** [comment]
That is caused by the Redactor plugin, which will be fixed in its next release (craftcms/redactor@c48df35092224853fce7ce030fd204333d12cc13).
